### PR TITLE
feat: add verbose support for tql explain/analyze

### PR DESCRIPTION
**Pull request by etolbakov**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?
Add support of `VERBOSE` for `TQL EXPLAIN` And `TQL ANALYZE` to get more detailed output 

## Checklist

- [x]  I have written the necessary rustdoc comments.
- [x]  I have added the necessary unit tests and integration tests.
- [x]  This PR does not require documentation updates.

## Refer to a related PR or issue link (optional)
https://github.com/GreptimeTeam/greptimedb/issues/2850

### CODE REVIEW

**Comment by etolbakov** [comment]
I haven't pushed the change for sqlness tests as I still playing with `SQLNESS REPLACE` declarations

<details>
  <summary>TQL EXPLAIN VERBOSE (0, 10, '5s') test;</summary> 
  ```txt
+------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
| plan_type                                                  | plan                                                                                                           |
+------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
| initial_logical_plan                                       | PromInstantManipulate: range=[0..0], lookback=[300000], interval=[300000], time index=[j]                      |
|                                                            |   PromSeriesNormalize: offset=[0], time index=[j], filter NaN: [false]                                         |
|                                                            |     PromSeriesDivide: tags=["k"]                                                                               |
|                                                            |       Sort: test.k DESC NULLS LAST, test.j DESC NULLS LAST                                                     |
|                                                            |         Filter: test.j >= TimestampMillisecond(-300000, None) AND test.j <= TimestampMillisecond(300000, None) |
|                                                            |           TableScan: test                                                                                      |
| logical_plan after count_wildcard_rule                     | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after StringNormalizationRule                 | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after inline_table_scan                       | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after type_coercion                           | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after DistPlannerAnalyzer                     | PromInstantManipulate: range=[0..0], lookback=[300000], interval=[300000], time index=[j]                      |
|                                                            |   PromSeriesNormalize: offset=[0], time index=[j], filter NaN: [false]                                         |
|                                                            |     PromSeriesDivide: tags=["k"]                                                                               |
|                                                            |       MergeScan [is_placeholder=false]                                                                         |
| analyzed_logical_plan                                      | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after eliminate_nested_union                  | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after simplify_expressions                    | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after unwrap_cast_in_comparison               | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after replace_distinct_aggregate              | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after eliminate_join                          | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after decorrelate_predicate_subquery          | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after scalar_subquery_to_join                 | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after extract_equijoin_predicate              | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after simplify_expressions                    | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after merge_projection                        | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after rewrite_disjunctive_predicate           | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after eliminate_duplicated_expr               | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after eliminate_filter                        | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after eliminate_cross_join                    | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after common_sub_expression_eliminate         | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after eliminate_limit                         | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after propagate_empty_relation                | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after eliminate_one_union                     | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after filter_null_join_keys                   | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after eliminate_outer_join                    | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after push_down_limit                         | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after push_down_filter                        | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after single_distinct_aggregation_to_group_by | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after simplify_expressions                    | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after unwrap_cast_in_comparison               | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after common_sub_expression_eliminate         | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after push_down_projection                    | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after eliminate_projection                    | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after push_down_limit                         | SAME TEXT AS ABOVE                                                                                             |
| logical_plan after OrderHintRule                           | SAME TEXT AS ABOVE                                                                                             |
| logical_plan                                               | PromInstantManipulate: range=[0..0], lookback=[300000], interval=[300000], time index=[j]                      |
|                                                            |   PromSeriesNormalize: offset=[0], time index=[j], filter NaN: [false]                                         |
|                                                            |     PromSeriesDivide: tags=["k"]                                                                               |
|                                                            |       MergeScan [is_placeholder=false]                                                                         |
| initial_physical_plan                                      | PromInstantManipulateExec: range=[0..0], lookback=[300000], interval=[300000], time index=[j]                  |
|                                                            |   PromSeriesNormalizeExec: offset=[0], time index=[j], filter NaN: [false]                                     |
|                                                            |     PromSeriesDivideExec: tags=["k"]                                                                           |
|                                                            |       MergeScanExec: peers=[6171868004352(1437, 0), ]                                                          |
|                                                            |                                                                                                                |
| physical_plan after OutputRequirements                     | OutputRequirementExec                                                                                          |
|                                                            |   PromInstantManipulateExec: range=[0..0], lookback=[300000], interval=[300000], time index=[j]                |
|                                                            |     PromSeriesNormalizeExec: offset=[0], time index=[j], filter NaN: [false]                                   |
|                                                            |       PromSeriesDivideExec: tags=["k"]                                                                         |
|                                                            |         MergeScanExec: peers=[6171868004352(1437, 0), ]                                                        |
|                                                            |                                                                                                                |
| physical_plan after aggregate_statistics                   | SAME TEXT AS ABOVE                                                                                             |
| physical_plan after join_selection                         | SAME TEXT AS ABOVE                                                                                             |
| physical_plan after EnforceDistribution                    | OutputRequirementExec                                                                                          |
|                                                            |   PromInstantManipulateExec: range=[0..0], lookback=[300000], interval=[300000], time index=[j]                |
|                                                            |     RepartitionExec: partitioning=RoundRobinBatch(10), input_partitions=1                                      |
|                                                            |       PromSeriesNormalizeExec: offset=[0], time index=[j], filter NaN: [false]                                 |
|                                                            |         PromSeriesDivideExec: tags=["k"]                                                                       |
|                                                            |           MergeScanExec: peers=[6171868004352(1437, 0), ]                                                      |
|                                                            |                                                                                                                |
| physical_plan after CombinePartialFinalAggregate           | SAME TEXT AS ABOVE                                                                                             |
| physical_plan after EnforceSorting                         | OutputRequirementExec                                                                                          |
|                                                            |   PromInstantManipulateExec: range=[0..0], lookback=[300000], interval=[300000], time index=[j]                |
|                                                            |     RepartitionExec: partitioning=RoundRobinBatch(10), input_partitions=1                                      |
|                                                            |       PromSeriesNormalizeExec: offset=[0], time index=[j], filter NaN: [false]                                 |
|                                                            |         PromSeriesDivideExec: tags=["k"]                                                                       |
|                                                            |           SortExec: expr=[k@2 ASC NULLS LAST]                                                                  |
|                                                            |             MergeScanExec: peers=[6171868004352(1437, 0), ]                                                    |
|                                                            |                                                                                                                |
| physical_plan after coalesce_batches                       | SAME TEXT AS ABOVE                                                                                             |
| physical_plan after OutputRequirements                     | PromInstantManipulateExec: range=[0..0], lookback=[300000], interval=[300000], time index=[j]                  |
|                                                            |   RepartitionExec: partitioning=RoundRobinBatch(10), input_partitions=1                                        |
|                                                            |     PromSeriesNormalizeExec: offset=[0], time index=[j], filter NaN: [false]                                   |
|                                                            |       PromSeriesDivideExec: tags=["k"]                                                                         |
|                                                            |         SortExec: expr=[k@2 ASC NULLS LAST]                                                                    |
|                                                            |           MergeScanExec: peers=[6171868004352(1437, 0), ]                                                      |
|                                                            |                                                                                                                |
| physical_plan after PipelineChecker                        | SAME TEXT AS ABOVE                                                                                             |
| physical_plan after LimitAggregation                       | SAME TEXT AS ABOVE                                                                                             |
| physical_plan                                              | PromInstantManipulateExec: range=[0..0], lookback=[300000], interval=[300000], time index=[j]                  |
|                                                            |   RepartitionExec: partitioning=RoundRobinBatch(10), input_partitions=1                                        |
|                                                            |     PromSeriesNormalizeExec: offset=[0], time index=[j], filter NaN: [false]                                   |
|                                                            |       PromSeriesDivideExec: tags=["k"]                                                                         |
|                                                            |         SortExec: expr=[k@2 ASC NULLS LAST]                                                                    |
|                                                            |           MergeScanExec: peers=[6171868004352(1437, 0), ]                                                      |
|                                                            |                                                                                                                |
+------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
  ```
</details>

<details>
  <summary>TQL ANALYZE VERBOSE (0, 10, '5s') test;</summary>
  
  ### Heading
  ```txt
+------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| plan_type              | plan                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Plan with Metrics      | PromInstantManipulateExec: range=[0..10000], lookback=[300000], interval=[5000], time index=[j], metrics=[output_rows=4, elapsed_compute=46.419µs]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
|                        |   RepartitionExec: partitioning=RoundRobinBatch(10), input_partitions=1, metrics=[send_time=12.125µs, fetch_time=4.368625ms, repart_time=1ns]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
|                        |     PromSeriesNormalizeExec: offset=[0], time index=[j], filter NaN: [false], metrics=[output_rows=3, elapsed_compute=28.5µs]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
|                        |       PromSeriesDivideExec: tags=["k"], metrics=[output_rows=3, elapsed_compute=85ns]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
|                        |         SortExec: expr=[k@2 ASC NULLS LAST], metrics=[output_rows=3, elapsed_compute=1ns, spill_count=0, spilled_bytes=0]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
|                        |           MergeScanExec: peers=[6167573037056(1436, 0), ], metrics=[output_rows=3, greptime_exec_cost=0, finish_time=4.236917ms, ready_time=3.652958ms, first_consume_time=4.15725ms]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
|                        |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
| Plan with Full Metrics | PromInstantManipulateExec: range=[0..10000], lookback=[300000], interval=[5000], time index=[j], metrics=[start_timestamp{partition=4}=2024-02-26 21:40:12.568175 UTC, end_timestamp{partition=4}=2024-02-26 21:40:12.572745 UTC, elapsed_compute{partition=4}=42ns, output_rows{partition=4}=0, start_timestamp{partition=1}=2024-02-26 21:40:12.568206 UTC, end_timestamp{partition=1}=2024-02-26 21:40:12.572776 UTC, elapsed_compute{partition=1}=15.334µs, output_rows{partition=1}=2, start_timestamp{partition=0}=2024-02-26 21:40:12.568224 UTC, end_timestamp{partition=0}=2024-02-26 21:40:12.572776 UTC, elapsed_compute{partition=0}=30.668µs, output_rows{partition=0}=2, start_timestamp{partition=9}=2024-02-26 21:40:12.568229 UTC, end_timestamp{partition=9}=2024-02-26 21:40:12.572741 UTC, elapsed_compute{partition=9}=41ns, output_rows{partition=9}=0, start_timestamp{partition=6}=2024-02-26 21:40:12.568233 UTC, end_timestamp{partition=6}=2024-02-26 21:40:12.572754 UTC, elapsed_compute{partition=6}=41ns, output_rows{partition=6}=0, start_timestamp{partition=7}=2024-02-26 21:40:12.568247 UTC, end_timestamp{partition=7}=2024-02-26 21:40:12.572734 UTC, elapsed_compute{partition=7}=84ns, output_rows{partition=7}=0, start_timestamp{partition=2}=2024-02-26 21:40:12.568275 UTC, end_timestamp{partition=2}=2024-02-26 21:40:12.572750 UTC, elapsed_compute{partition=2}=1ns, output_rows{partition=2}=0, start_timestamp{partition=3}=2024-02-26 21:40:12.568302 UTC, end_timestamp{partition=3}=2024-02-26 21:40:12.572726 UTC, elapsed_compute{partition=3}=83ns, output_rows{partition=3}=0, start_timestamp{partition=8}=2024-02-26 21:40:12.568314 UTC, end_timestamp{partition=8}=2024-02-26 21:40:12.572736 UTC, elapsed_compute{partition=8}=42ns, output_rows{partition=8}=0, start_timestamp{partition=5}=2024-02-26 21:40:12.568318 UTC, end_timestamp{partition=5}=2024-02-26 21:40:12.572765 UTC, elapsed_compute{partition=5}=83ns, output_rows{partition=5}=0] |
|                        |   RepartitionExec: partitioning=RoundRobinBatch(10), input_partitions=1, metrics=[fetch_time{partition=0, inputPartition=4}=4.368625ms, repart_time{partition=0, inputPartition=4}=NOT RECORDED, send_time{partition=0, inputPartition=4}=12.125µs]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
|                        |     PromSeriesNormalizeExec: offset=[0], time index=[j], filter NaN: [false], metrics=[start_timestamp{partition=0}=2024-02-26 21:40:12.568322 UTC, end_timestamp{partition=0}=2024-02-26 21:40:12.572703 UTC, elapsed_compute{partition=0}=28.5µs, output_rows{partition=0}=3]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
|                        |       PromSeriesDivideExec: tags=["k"], metrics=[start_timestamp{partition=0}=2024-02-26 21:40:12.568323 UTC, end_timestamp{partition=0}=2024-02-26 21:40:12.572701 UTC, elapsed_compute{partition=0}=85ns, output_rows{partition=0}=3]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
|                        |         SortExec: expr=[k@2 ASC NULLS LAST], metrics=[start_timestamp{partition=0}=2024-02-26 21:40:12.568354 UTC, end_timestamp{partition=0}=2024-02-26 21:40:12.572621 UTC, elapsed_compute{partition=0}=NOT RECORDED, output_rows{partition=0}=3, spill_count{partition=0}=0, spilled_bytes{partition=0}=0]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
|                        |           MergeScanExec: peers=[6167573037056(1436, 0), ], metrics=[ready_time{partition=1}=3.652958ms, first_consume_time{partition=1}=4.15725ms, finish_time{partition=1}=4.236917ms, output_rows{partition=1}=3, greptime_exec_cost{partition=1}=0]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
|                        |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
| Output Rows            | 4                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
| Duration               | 4.652084ms                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
  ```
</details>

File: tests/cases/distributed/tql-explain-analyze/analyze.sql
Comment (waynexia) [inline tests/cases/distributed/tql-explain-analyze/analyze.sql:1]: Tip: you can put this under `cases/distributed/common/tql-explain-analyze`, where the `common` is a symlink shared between `distributed` and `standalone`. So these cases can be run in both environments.

**Review by waynexia — APPROVED** [review]
Looking great 🚀  The tests are also concrete, let's ignore the codecov since sqlness is not included.

File: tests/cases/distributed/tql-explain-analyze/analyze.sql
Comment (etolbakov) [reply tests/cases/distributed/tql-explain-analyze/analyze.sql:1]: ahhh... cool! Thanks @waynexia  noted!

File: src/sql/src/parsers/tql_parser.rs [hunk 1]
```diff
@@ -26,12 +26,14 @@ pub const TQL: &str = "TQL";
...
+const VERBOSE: &str = "VERBOSE";
+
 use sqlparser::parser::Parser;
 
 /// TQL extension parser, including:
 /// - `TQL EVAL <query>`
-/// - `TQL EXPLAIN <query>`
-/// - `TQL ANALYZE <query>`
+/// - `TQL EXPLAIN (VERBOSE) <query>`
+/// - `TQL ANALYZE (VERBOSE) <query>`
```
Comment (killme2008) [inline src/sql/src/parsers/tql_parser.rs:36]:
```suggestion
/// - `TQL EXPLAIN [VERBOSE] <query>`
/// - `TQL ANALYZE [VERBOSE] <query>`
```

To indicate that an item is optional, prefer to use the bracket notation `[item]`.

File: src/sql/src/statements/tql.rs [hunk 2]
```diff
@@ -29,20 +29,22 @@ pub struct TqlEval {
     pub query: String,
 }
 
-/// TQL EXPLAIN (like SQL EXPLAIN): doesn't execute the query but tells how the query would be executed.
+/// TQL EXPLAIN (VERBOSE) (like SQL EXPLAIN): doesn't execute the query but tells how the query would be executed.
```
Comment (killme2008) [inline src/sql/src/statements/tql.rs:32]:
```suggestion
/// TQL EXPLAIN [VERBOSE] (like SQL EXPLAIN): doesn't execute the query but tells how the query would be executed.
```

File: src/sql/src/statements/tql.rs [hunk 3]
```diff
@@ -29,20 +29,22 @@ pub struct TqlEval {
...
 pub struct TqlExplain {
     pub start: String,
     pub end: String,
     pub step: String,
     pub query: String,
+    pub is_verbose: bool,
 }
 
-/// TQL ANALYZE (like SQL ANALYZE): executes the plan and tells the detailed per-step execution time.
+/// TQL ANALYZE (VERBOSE) (like SQL ANALYZE): executes the plan and tells the detailed per-step execution time.
```
Comment (killme2008) [inline src/sql/src/statements/tql.rs:42]:
```suggestion
/// TQL ANALYZE [VERBOSE] (like SQL ANALYZE): executes the plan and tells the detailed per-step execution time.
```

**Review by killme2008 — APPROVED** [review]
Looks great! 

There are a few suggestions attached, but they are not blocking.

**Pull request merged**